webserver/views/datasets.py

```diff
@@ -308,7 +308,11 @@ def edit(dataset_id):
 def delete(dataset_id):
     ds = get_dataset(dataset_id)
     if ds["author"] != current_user.id:
-        raise Unauthorized("You can't delete this dataset.")
+        return jsonify(
```

Comment:
I'd rather not.

1. This is not an API.
2. If the way API exceptions output content changes, this might break the UI (and this is the only place this endpoint is used). So I'd rather keep it more explicit here than try to find mysterious bugs in future.